src/Cake.Common.Tests/Unit/IO/FileAliasesTests.cs

```diff
@@ -1129,5 +1129,86 @@ public void Should_Return_Absolute_Directory_Path()
                 Assert.Equal("/Working/build.txt", result.FullPath);
             }
         }
+
+        public sealed class TheFileSizeMethod
+        {
+            [Fact]
+            public void Should_Throw_If_Context_Is_Null()
+            {
+                // Given, When
+                var result = Record.Exception(() => FileAliases.FileSize(null, "some file"));
+
+                // Then
+                Assert.IsArgumentNullException(result, "context");
+            }
+
+            [Fact]
+            public void Should_Throw_If_Path_Is_Null()
+            {
+                // Given
+                var context = Substitute.For<ICakeContext>();
+
+                // When
+                var result = Record.Exception(() => FileAliases.FileSize(context, null));
+
+                // Then
+                Assert.IsArgumentNullException(result, "filePath");
+            }
+
+            [Fact]
+            public void Should_Return_NegativeOne_If_Directory_Does_Not_Exist()
+            {
+                // Given
+                var context = Substitute.For<ICakeContext>();
+                var environment = FakeEnvironment.CreateUnixEnvironment();
+                var fileSystem = new FakeFileSystem(environment);
+                context.FileSystem.Returns(fileSystem);
+                context.Environment.Returns(environment);
+
+                // When
+                var result = FileAliases.FileSize(context, "non-existent-file.txt");
+
+                // Then
+                Assert.Equal(result, -1);
+            }
+
+            [Fact]
```

Comment:
This test is identical to `Should_Return_Size_If_Absolute_Path_Exist()`. Can we make this a theory instead since the  only thing that differ is the path (absolute or relative)?